testdata/scripts/node/validate/invalid-ocr-p2p.txtar

```diff
@@ -290,6 +290,11 @@ DeltaDial = '15s'
 DeltaReconcile = '1m0s'
 ListenAddresses = []
 
+[Capabilities.Registry]
+RemoteAddress = ''
+NetworkID = 'evm'
+ChainID = '1'
+
```

Comment:
Why was this needed? If capabilities aren't enabled, this shouldn't be required 🤔